collector/textfile.go

```diff
@@ -13,8 +13,8 @@ import (
 
 	dto "github.com/prometheus/client_model/go"
 
-	"code.google.com/p/goprotobuf/proto"
 	"github.com/golang/glog"
+	"github.com/golang/protobuf"
```

Comment:
Thanks!

1) This needs to be `github.com/golang/protobuf/proto`, because the root of the repo doesn't actually contain any Go packages. This is the cause of the Travis build error.

2) The import grouping (i.e. empty line between 17/18) was fine before, so I'd keep it. We're admittedly over-engineering our import groups too much, but currently we do the following groupings: a) stdlib imports, b) renamed stdlib imports, c) repo-external imports, d) renamed repo-external imports, e) intra-repo imports, f) renamed intra-repo imports.